libfwupdplugin/fu-bios-settings.c

```diff
@@ -582,7 +584,7 @@ fu_bios_settings_to_variant(FuBiosSettings *self, gboolean trusted)
  * Since: 1.8.4
  **/
 gboolean
-fu_bios_settings_from_json(FuBiosSettings *self, JsonNode *json_node, GError **error)
+fu_bios_settings_from_json_node(FuBiosSettings *self, JsonNode *json_node, GError **error)
```

Comment:
I think the original name was better -- it matches the others, e.g. fwupd_security_attr_from_json

Reply:
ack